spec/factories/ext_management_system.rb

```diff
@@ -167,4 +173,12 @@
       x.authentications << FactoryGirl.create(:authentication)
     end
   end
+
+  # Leaf classes for middleware_manager
+
+  factory :middleware_manager,
```

Comment:
the factory is hawkular specific, so I would name it accordingly, so when used from tests it's clear which exact type of provider is tested